openshift/patches/0002-rust-manifest.patch

```diff
@@ -0,0 +1,8 @@
+diff --git a/templates/rust/manifest.yaml b/templates/rust/manifest.yaml
```

Comment:
There is no rust template modification in midstream. I believe this should be fixed in upstream code.

Reply:
Hmm - not sure about that. In upstream manifest.yaml, there is only the buildpack set and this should be removed in downstream (not modified - just removed entirely)

Reply:
You are probably right.